test/container_test.py

```diff
@@ -834,6 +839,30 @@ def test_checkpoint_and_restore_success(unix_servicer, event_loop):
     assert _unwrap_scalar(ret) == 42**2
 
 
+
+@skip_windows_unix_socket
+def test_error_open_connection(unix_servicer, event_loop):
+    """Functions fail to checkpoint if connections are open."""
+    with pytest.raises(ConnectionError):
+        _run_container(
+            unix_servicer,
+            "modal_test_support.functions",
+            "CheckpointingClsNetworkConnectionOpen.open_connection",
+            is_checkpointing_function=True,
+        )
+        assert not any(isinstance(request, api_pb2.ContainerCheckpointRequest) for request in unix_servicer.requests)
```

Comment:
Should this line be outside the `with` block? When will it run?

Reply:
This is wrong. Just removed.